tool/tctl/common/sessions_command.go

```diff
@@ -0,0 +1,88 @@
+/*
+Copyright 2022 Gravitational, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package common
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"time"
+
+	"github.com/gravitational/kingpin"
+	"github.com/gravitational/trace"
+
+	"github.com/gravitational/teleport"
+	"github.com/gravitational/teleport/api/types"
+	"github.com/gravitational/teleport/lib/auth"
+	"github.com/gravitational/teleport/lib/defaults"
+	"github.com/gravitational/teleport/lib/service"
+	"github.com/gravitational/teleport/tool/common"
+	tctlcommon "github.com/gravitational/teleport/tool/tctl/common"
+)
+
+// SessionsCommand implements "tctl sessions" group of commands.
+type SessionsCommand struct {
+	config *service.Config
+
+	// format is the output format (text, json, or yaml)
+	format string
+	// sessionsList implements the "tctl sessions ls" subcommand.
+	sessionsList *kingpin.CmdClause
+	// fromUTC is the start time to use for the range of sessions listed by the recorded session listing command
+	fromUTC string
+	// toUTC is the start time to use for the range of sessions listed by the recorded session listing command
+	toUTC string
+	// sessionsPerPage is the maximum number of sessions to show oer page of results
+	sessionsPerPage int
+}
+
+// Initialize allows SessionsCommand to plug itself into the CLI parser
+func (c *SessionsCommand) Initialize(app *kingpin.Application, config *service.Config) {
+	c.config = config
+
+	sessions := app.Command("sessions", "Operate on recorded sessions.")
+	c.sessionsList = sessions.Command("ls", "List recorded sessions.")
+	c.sessionsList.Flag("format", "Output format, 'text', 'json', or 'yaml'. Defaults to 'text'.").Default(teleport.Text).StringVar(&c.format)
+	c.sessionsList.Flag("from-utc", fmt.Sprintf("Start of time range in which sessions are listed. Format %s. Defaults to 24 hours ago.", time.RFC3339)).StringVar(&c.fromUTC)
+	c.sessionsList.Flag("to-utc", fmt.Sprintf("End of time range in which sessions are listed. Format %s. Defaults to current time.", time.RFC3339)).StringVar(&c.toUTC)
+	c.sessionsList.Flag("limit", fmt.Sprintf("Maximum number of sessions per page to show. Default %s.", defaults.DefaultSearchSessionPageLimit)).Default(defaults.DefaultSearchSessionPageLimit).IntVar(&c.sessionsPerPage)
```

Comment:
What does per-page mean in this context? Are you implementing paging in the CLI?

Reply:
Ah you are right.
This doesn't really make much sense as a message anymore now that the limiting is done through the --limit